src/NUnitEngine/nunit.engine/Extensibility/ExtensionAssembly.cs

```diff
@@ -0,0 +1,77 @@
+﻿// ***********************************************************************
```

Comment:
This file has been moved and isn't entirely new - I guess GitHub hasn't noticed as there's also some changes.